converters/fromZigbee.js

```diff
@@ -5556,6 +5556,51 @@ const converters = {
             return result;
         },
     },
+    schneider_ui_action: {
+        cluster: 'wiserDeviceInfo',
+        type: 'attributeReport',
+        convert: (model, msg, publish, options, meta) => {
+            if (hasAlreadyProcessedMessage(msg)) return;
+
+            const data = msg.data['deviceInfo'].split(',');
+            if(data[0] === 'UI' && data[1]) {
+                publish({action: data[1]});
```

Comment:
instead of calling publish here, do `const result = {action: data[1]};` and replace `publish({occupied_heating_setpoint: occupiedHeatingSetpoint/100});` with:

```js
result.occupied_heating_setpoint = occupiedHeatingSetpoint/100;
return result;
```

Reply:
Fixed